### PR TITLE
docs: add static documentation website

### LINKED ISSUE #113: [LLM][FRONT] Implement the design of the chosen mockup to deploy the site.
# Context

# Procedure

# Acceptance

Étant  donné \<contexte\>, lorsque \<action\> alors \<résultat attendu\>.

---

### DoR

- [ ] Adhérences Identifiées
- [ ] Synchronisation Partenaires
- [ ] Maquettes / Zoning Disponibles
- [ ] Démo à prévoir

___

> responsible: xxx 
> 
> reviewer: xxx

### PR DESCRIPTION
**Pull request by white-gorilla**
This PR adds the static documentation site, customized with a landing page.
On this landing page, the Highlights section (benchmarks + schema) is missing.

`Note`: For the moment, I have left some things as comments in the configuration file.

#### Reminder

- The two commands to launch the preview and the generation are in the file `docs/readme.md`.
- Don't forget to review the Github Action part.

#### Tested browsers

- brave: v1.45.113  chromium 107.0.5304.62
- Safari desktop: 16.1 (17614.2.9.1.13, 17614)
- Firefox desktop: 107.0b5 (64 bits)
- Firefox mobile: 106.0 (20303). iPad Pro / iPhone 13
- Safari mobile: iOS 16.1. iPad Pro / iPhone 13

fix #113

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Kernl static documentation site
+
+## Preview the static site locally 
+
+The easiest and least intrusive way is to use docker.
+
+```shell
+# Installation
```
Comment (gaetansnl) [inline docs/README.md:8]: not useful, image will be pulled anyway

File: docs/overrides/home.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+                    <svg width="25" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M11.4 2.1c-4.6.5-8.3 4.2-8.8 8.7-.5 4.7 2.2 8.9 6.3 10.5.3.1.6-.1.6-.5v-1.6s-.4.1-.9.1c-1.4 0-2-1.2-2.1-1.9-.1-.4-.3-.7-.6-1-.3-.1-.4-.1-.4-.2 0-.2.3-.2.4-.2.6 0 1.1.7 1.3 1 .5.8 1.1 1 1.4 1 .4 0 .7-.1.9-.2.1-.7.4-1.4 1-1.8-2.3-.5-4-1.8-4-4 0-1.1.5-2.2 1.2-3-.1-.2-.2-.7-.2-1.4 0-.4 0-1 .3-1.6 0 0 1.4 0 2.8 1.3.5-.2 1.2-.3 1.9-.3s1.4.1 2 .3C15.8 6 17.3 6 17.3 6c.2.6.2 1.2.2 1.6 0 .8-.1 1.2-.2 1.4.7.8 1.2 1.8 1.2 3 0 2.2-1.7 3.5-4 4 .6.5 1 1.4 1 2.3v2.6c0 .3.3.6.7.5 3.7-1.5 6.3-5.1 6.3-9.3 0-6-5.1-10.7-11.1-10Z"/>
+                    </svg>
+                    <span>Contribute</span>
+                </a>
+            </div>
+        </div>
+    </section>
+    <section class="knl-snippet md-grid">
+        <pre><code class="knl-snippet__code"><span class="instruction">from</span> transformers <span
```
Comment (gaetansnl) [inline docs/overrides/home.html:56]: we changed the way it works, need to change the example

File: docs/overrides/home.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
+{% extends "main.html" %}
```
Comment (gaetansnl) [inline docs/overrides/home.html:1]: Still not fan of centering, explained here 
https://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/
https://thewebsitearchitect.com/does-center-aligned-text-matter-for-accessibility/

File: docs/overrides/home.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+{% block tabs %}
+{{ super() }}
+<style>
```
Comment (gaetansnl) [inline docs/overrides/home.html:4]: why is isn't in css ?

File: .github/workflows/deploy-static-site.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
+      - run: mkdocs build
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          # Upload entire repository
```
Comment (gaetansnl) [inline .github/workflows/deploy-static-site.yml:40]: entire repository ? what to you mean

**Review by gaetansnl — APPROVED** [review]
nice work, congrats 🎉 The only thing I'm not sure is the workflow, never used github pages

The important comment is about the code example, we changed things yesterday

File: docs/README.md (hunk 1 above)
Comment (white-gorilla) [reply docs/README.md]: `done`: I deleted the unnecessary line.

File: docs/overrides/home.html (hunk 2 above)
Comment (white-gorilla) [reply docs/overrides/home.html:56]: `done`: the snippet is iso with the readme file of kernl.

File: docs/overrides/home.html (hunk 3 above)
Comment (white-gorilla) [reply docs/overrides/home.html:1]: You are absolutely right. It's much less readable and accessible. But visually it becomes different.
I would rather readjust and have it validated by a feedback from the team. I note it and make an issue in this sense.

File: docs/overrides/home.html (hunk 4 above)
Comment (white-gorilla) [reply docs/overrides/home.html:4]: This is due to the context of Material for MkDocs. For styles to be applied, they must be at the bottom of the cascading.

File: .github/workflows/deploy-static-site.yml (hunk 5 above)
Comment (white-gorilla) [reply .github/workflows/deploy-static-site.yml:40]: `done`: upload entire static site

**Pull request merged**